_data/wildfly-categories.yaml

```diff
@@ -0,0 +1,140 @@
+---
+categories:
+  - name: Agroal
+    id: agroal
+    description: Agroal Data Source
+  - name: Batch
+    id: batch
+    description: Jakarta EE Batch
+  - name: Bean Validation
+    id: bean-validation
+    description: Jakarta EE Bean Validation
+  - name: Bootable Jar
+    id: bootable-jar
+    description: Bootable Jar
+  - name: Command Line Interface (CLI)
+    id: cli
+    description: Command Line Interface
+  - name: Cloud
+    id: cloud
+    description: Developing and Running a WildFly application on the cloud
+  - name: User Experience
+    id: user-experience
+    description: Impact how the user experiences working with WildFly
+  - name: Clustering
+    id: clustering
+    description: Clustering, including Infinispan and JGroups
+  - name: Concurrency
+    id: concurrency
+    description: Concurrency
+  - name: Console
+    id: console
+    description: Console
+  - name: WildFly Controller
```

Comment:
Let's drop this one and just use 'core' for the 3 proposals.

Reply:
done in 7178388f0c5443e3a2811b3ebfbe937533584b9a